decidim-assemblies/app/views/decidim/assemblies/admin/assemblies/_assembly_row.html.erb

```diff
@@ -0,0 +1,91 @@
+<tr>
+  <td>
+    <% if assembly.promoted? %>
+      <%= icon_with_tooltip "star-s-fill", t("models.assembly.fields.promoted", scope: "decidim.admin") %>
+    <% end %>
+
+    <% if allowed_to? :update, :assembly, assembly: assembly %>
+      <%= link_to translated_attribute(assembly.title), edit_assembly_path(assembly) %>
+    <% elsif allowed_to? :read, :component, assembly: assembly %>
+      <%= link_to translated_attribute(assembly.title), components_path(assembly) %><br>
+    <% else %>
+      <%= translated_attribute(assembly.title) %>
+    <% end %>
+  </td>
+  <td class="table-list__date">
+    <%= l(assembly.created_at, format: :short) %>
+  </td>
+  <td class="table-list__state">
+    <% if assembly.private_space? %>
+      <span><%= t("assemblies.index.private", scope: "decidim.admin") %></span>
+    <% else %>
+      <span><%= t("assemblies.index.public", scope: "decidim.admin") %></span>
+    <% end %>
+  </td>
+  <td class="table-list__state">
+    <% if assembly.published? %>
+      <span class="label success !text-sm"><%= t("assemblies.index.published", scope: "decidim.admin") %></span>
+    <% else %>
+      <span class="label alert !text-sm"><%= t("assemblies.index.unpublished", scope: "decidim.admin") %></span>
+    <% end %>
+  </td>
+  <td class="table-list__actions">
+    <% if view == :deleted %>
+      <% if allowed_to? :restore, :assembly, trashable_deleted_resource: assembly %>
+        <%= icon_link_to "refresh-line", restore_assembly_path(assembly), t("actions.restore", scope: "decidim.admin"), method: :patch, class: "action-icon--restore" %>
+      <% else %>
+        <span class="action-space icon"></span>
+      <% end %>
+    <% else %>
+      <% if allowed_to? :read, :share_tokens, current_participatory_space: assembly %>
+        <%= icon_link_to "share-line", decidim_admin_assemblies.assembly_share_tokens_path(assembly), t("actions.share_tokens", scope: "decidim.admin"), class: "action-icon--new" %>
+      <% else %>
+        <span class="action-space icon"></span>
+      <% end %>
+      <% if allowed_to? :update, :assembly, assembly: assembly %>
+        <%= icon_link_to "pencil-line", edit_assembly_path(assembly), t("actions.configure", scope: "decidim.admin"), class: "action-icon--new" %>
+      <% else %>
+        <span class="action-space icon"></span>
+      <% end %>
+      <% if assembly.children.count.positive? || allowed_to?(:read, :assembly, assembly:) %>
+        <%= icon_link_to "government-line",
+                         url_for(query_params_with(parent_id_eq: assembly.id)),
+                         t("decidim.admin.titles.assemblies"),
+                         class: "action-icon--dial #{"highlighted" if assembly.children.count.positive?}" %>
+      <% else %>
+        <span class="action-space icon"></span>
+      <% end %>
+      <% if allowed_to? :copy, :assembly, assembly: assembly, assembly: parent_assembly %>
```

Comment:
@antopalidi which is the correct argument here? The `assembly` argument is duplicated.